ui/app/AppLayouts/Wallet/WalletUtils.qml

```diff
@@ -73,4 +100,51 @@ QtObject {
 
         return value - Math.max(0.0001, Math.min(0.01, value * 0.1))
     }
+
+    function getAssetForSendTx(tx) {
+        if (tx.isNFT) {
+            return {
+                uid: tx.tokenID,
+                chainId: tx.chainId,
+                name: tx.nftName,
+                imageUrl: tx.nftImageUrl,
+                collectionUid: "",
+                collectionName: ""
+            }
+        } else {
+            return tx.symbol
+        }
+    }
+
+    function isTxRepeatable(tx) {
+        if (!tx || tx.txType !== Constants.TransactionType.Send)
+            return false
+
+        let res = root.lookupAddressObject(tx.sender)
+        if (!res || res.type !== RootStore.LookupType.Account || res.object.walletType == Constants.watchWalletType)
```

Comment:
This enum (`RootStore.LookupType.Account`) is coming from the `RootStore`, makes no sense here